src/librustdoc/clean/types.rs

```diff
@@ -764,7 +764,7 @@ impl Item {
         Some(tcx.visibility(def_id))
     }
 
-    pub(crate) fn attributes_without_repr(&self, tcx: TyCtxt<'_>, is_json: bool) -> Vec<String> {
+    fn attributes_without_repr(&self, tcx: TyCtxt<'_>, is_json: bool) -> Vec<String> {
```

Comment:
This one is actually only called by `attributes_with_repr` IIRC correctly. In https://github.com/rust-lang/rust/pull/116882 I actually inline this method and rename `atrributes_with_repr` to `attributes` because repr is an attribute and since the relevant bit of Guillaume's ADT / type alias unification PR is the fact that we now have a standalone fn `repr_attributes`